### ISSUE TITLE
Transfer Learning

### ISSUE THREAD

**Issue by slavakurilyak** [produvia/ai-platform#48]
# Goal(s)

- Transfer knowledge from a well-labeled domain to a similar but different domain with limited or no labels

# Input(s)

- Model weights trained on one task

# Output(s)

- Model weights trained on another task

# Objective Function(s)

- TBD

**Comment by p9anand** [comment]
I have used CRAFT network in my current company for OCR project. i can integrate the same if you want.

**Comment by slavakurilyak** [comment]
@p9anand Can you share a link to the CRAFT network you mentioned?

**Comment by slavakurilyak** [comment]
Initial work by @cfmusoles: #90

**Comment by p9anand** [comment]
https://github.com/clovaai/CRAFT-pytorch

**Comment by slavakurilyak** [comment]
@p9anand Feel free to upload an implementation of CRAFT under the following folder: 

[tasks/computer-vision/scene-text-detection](https://github.com/produvia/ai-platform/tree/master/tasks/computer-vision/scene-text-detection)

You can use PyTorch or other implementations of CRAFT, including:

1. https://github.com/guruL/Character-Region-Awareness-for-Text-Detection-
2. https://github.com/autonise/CRAFT-Remade
3. https://github.com/backtime92/CRAFT-Reimplementation
4. https://github.com/namedysx/CRAFT-tensorflow
5. https://github.com/inouetaka/CRAFT

While CRAFT achieves state-of-the-art on [MSRA-TD500](http://www.iapr-tc11.org/mediawiki/index.php/MSRA_Text_Detection_500_Database_(MSRA-TD500)) dataset, consider training your model on other datasets:

1. [CTW](https://ctwdataset.github.io)
2. [ICDAR 2019 Robust Reading Challenge](https://rrc.cvc.uab.es/?ch=11)

Cheers!
